### ISSUE TITLE
Missing lead time encoding?

### ISSUE THREAD

**Issue by ValterFallenius** [openclimatefix/metnet#28]
**Too small changes depending on lead time**
The model is able to learn something but the output image seems to change too little depending on the lead time encoding (one-hot from input layer). Here are some examples of the output from 2 different models, one with 60 leadtimes and one with only 8. The left hand plots show the ground truth precipitation in the prediction zone at different lead times, the right hand side shows P(rain_rate>0.2 mm/h) which means I sum the softmax probabillites of all the 127 bins corresponding to rain>0.2mm/h.

**60 lead times (5,10,15... 300 min)**

<img width="817" alt="3 leadtimes,8" src="https://user-images.githubusercontent.com/21970939/162745659-28c38ad3-2e00-4710-8950-7d495f01300d.PNG">
<img width="797" alt="3 leadtimes,2" src="https://user-images.githubusercontent.com/21970939/162745723-4d80fd74-7891-4882-903f-baebb439304c.PNG">
<img width="819" alt="3 leadtimes" src="https://user-images.githubusercontent.com/21970939/162745788-14c483c2-0142-412a-b705-be526c8b1d21.PNG">

**Only 8 leadtimes (15,30,45... 120 minutes):**
(I changed the cmap to make it clearer that the two plots are not plotting the same thing).

<img width="774" alt="nowcasting1" src="https://user-images.githubusercontent.com/21970939/162746023-5f75b136-f65d-400b-8d94-21d939a73d5f.PNG">
<img width="773" alt="nowcasting2" src="https://user-images.githubusercontent.com/21970939/162746458-bdddfa82-1475-4435-9d28-b94251c1142b.PNG">

For full 60 lead time network check out:  [w&b, 60 leads](https://wandb.ai/valterfallenius/lit-wandb/reports/60-lead-time-network--VmlldzoxODIyOTkx?accessToken=s9xe8n42i41tx9jmvi6lyd2vbgnpz0zhm69kv32bmz45fy2n5sq3r81tzw100s12)

For 8 lead time network check out: [w&b, 8 leads](https://wandb.ai/valterfallenius/lit-wandb/reports/Model-with-8-leadtimes--VmlldzoxODIyOTA2?accessToken=ypfk4tkvn5lgn1fsk3vw71inh6z7tiv1qbnkc7bct45zxfyu4a30j58s9ns0e390)

Should be noted that the 8 lead time network has not yet started overfitting.

I have implemented a sampling quality pass that during makes sure each training sample only samples a lead time when there is at least 5 rain pixels. 

I am suspecting the axial attention layer again as a bottleneck. Maybe I'm not using it right. We added a positional embedding so that it would know which pixel was where in the input layer, I was wondering if we should add an embedding for which channel it is looking at. Since the model seems to be forgetting which lead time it is handling the ConvGRU spits out 256x28x28 tensor.

Why is it performing so poorly?
